Explorer/Assets/DCL/WebRequests/WebRequestController.cs

```diff
@@ -14,22 +18,81 @@ public class WebRequestController : IWebRequestController
         private readonly IWebRequestsAnalyticsContainer analyticsContainer;
         private readonly IWeb3IdentityCache web3IdentityCache;
         private readonly IRequestHub requestHub;
+        private readonly string csvPath;
+        private static readonly object csvLock = new object();
 
-        public WebRequestController(IWebRequestsAnalyticsContainer analyticsContainer, IWeb3IdentityCache web3IdentityCache, IRequestHub requestHub)
+        public WebRequestController(
+            IWebRequestsAnalyticsContainer analyticsContainer,
+            IWeb3IdentityCache web3IdentityCache,
+            IRequestHub requestHub)
         {
             this.analyticsContainer = analyticsContainer;
             this.web3IdentityCache = web3IdentityCache;
             this.requestHub = requestHub;
+
+            // Set up CSV file path in persistent data path
+            csvPath = Path.Combine(Application.persistentDataPath, "request_metrics.csv");

```

Comment:
Where is the code that checks some command line argument or feature flag before doing this? We don't want to litter users' computers with CSV files, do we?

Reply:
Also, code to make sure the CSV file does not grow forever, etc, etc.